FormHandler/AbstractMessageFormHandler.php

```
@@ -84,4 +98,17 @@ protected function getAuthenticatedParticipant()
     {
         return $this->participantProvider->getAuthenticatedParticipant();
     }
+
+    /**
+     * BC layer to retrieve the current request directly or from a stack.
+     *
+     * @return null|Request
```

If `null` it should throw an exception instead as the class doesn't support it.